@navikt/internal/css/timeline.css

```
@@ -0,0 +1,222 @@
+.navdsi-timeline {
+  position: relative;
+  display: grid;
+  grid-template-columns: auto minmax(0, 1fr);
+  width: 100%;
+}
+
+.navdsi-timeline__axislabels {
+  position: relative;
+  height: 1rem;
+  box-sizing: content-box;
+  margin-bottom: 16px;
+  grid-column: 2;
+  margin-left: 2.375rem;
+}
+
+.navdsi-timeline__row__label {
+  white-space: nowrap;
+  width: auto;
+  margin-right: 1.125rem;
+  grid-column: 1;
+  font-size: var(--navds-font-size-medium);
+  height: 1.5rem;
+  display: flex;
+  align-items: center;
+  gap: var(--navds-spacing-4);
+}
+
+.navdsi-timeline__axislabels > * {
+  position: absolute;
+  font-size: var(--navds-font-size-small);
+  color: var(--navds-global-color-gray-600);
+}
+
+.navdsi-timeline__row {
+  display: flex;
+  background: var(--navds-global-color-gray-50);
+  margin: 0 0 var(--navds-spacing-5);
+  padding-left: 2.375rem;
+  grid-column: 2;
+}
+
+.navdsi-timeline__row:last-of-type {
+  margin-bottom: 0;
+}
+
+.navdsi-timeline__row__periods {
+  height: var(--navds-font-line-height-large);
+  position: relative;
+  width: 100%;
+  margin: 0;
+}
+
+.navdsi-timeline__row__periods li {
```

Tror popover-dialog kan ende opp inne i `li`-elementet som da denne kan treffe
```suggestion
.navdsi-timeline__row__periods > li {
```